### ISSUE TITLE
Incorrectly Identifies some binary files as code or text

### ISSUE THREAD

**Issue by noway** [google/magika#65]
I've generated 2435 files that are classified as text and code, although under manual inspection the files are binary.

Here is the repo with the generated files here: https://github.com/noway/magika-binary-misclassification-as-text

I've used fuzzing. [More info here](https://github.com/noway/magika-binary-misclassification-as-text/blob/main/METHODOLOGY.md).

Cheers!

**Comment by devilkadabra69** [comment]
.bin file is not listed in [supported files](https://github.com/google/magika/blob/main/docs/supported-content-types-list.md)

**Comment by reyammer** [comment]
Yep, random stream of bytes should fall under `unknown`. Thanks @noway for reporting and for the fuzzing, interesting stuff. Marked this issue with proper tags so that we remember about it when we plan for next iterations. Thanks!
